src/timbre_json_appender/core.clj

```diff
@@ -205,22 +204,24 @@
   `key-names`: Map of log key names. Can be used to override the default key names in `default-key-names`"
   ([]
    (install nil))
-  ([{:keys [level min-level pretty inline-args? level-key msg-key should-log-field-fn ex-data-field-fn key-names]
-     :or {level-key           :level
-          pretty              false
+  ([{:keys [level min-level pretty inline-args? level-key msg-key should-log-field-fn ex-data-field-fn key-names collect-vargs-fn]
+     :or {pretty              false
           inline-args?        true
-          msg-key             :msg
           should-log-field-fn default-should-log-field-fn
           ex-data-field-fn    default-ex-data-field-fn
-          key-names           default-key-names}}]
+          key-names           default-key-names
+          msg-key             (:msg key-names)
+          level-key           (:level key-names)
```

Comment:
Get defaults for msg-key and level-key from key-names which falls back to default-key-names